package-lock.json

```diff
@@ -2161,7 +2172,7 @@
     },
     "optimist": {
       "version": "0.6.1",
-      "resolved": "https://registry.npmjs.org/optimist/-/optimist-0.6.1.tgz",
+      "resolved": "https://artifactory.secureserver.net/artifactory/api/npm/node-virt/optimist/-/optimist-0.6.1.tgz",
```

Comment:
Looks like internal `.npmconfig` leaked into this

Reply:
Good catch. Fixed and added an `.npmrc` to here to help avoid this sort of thing.